### ISSUE TITLE
Regarding Dogon literature

### ISSUE THREAD

**Issue by bambooforest** [dogonlanguages/meta#4]
todo: go through the literature and archive / store pertinent files not in Zenodo, LDH, Deepblue:

https://github.com/clld/dogonlanguages-data/tree/master/beta/literature

regarding much of the grammars, texts, etc., they are published online in typically two places:

* MPG.PuRe
* UMich's Deepblue repository

e.g.

* https://pure.mpg.de/pubman/faces/ViewItemOverviewPage.jsp?itemId=item_2397771
* https://deepblue.lib.umich.edu/handle/2027.42/139022

which is reflected in the .bib file:

https://github.com/clld/dogonlanguages-data/blob/master/beta/Dogon.bib#L2391-L2403

and to LDH:

https://ldh.clld.org/sources/item_2397771

**Comment by xrotwang** [comment]
As soon as we have the bibliography in a repository that is released to Zenodo, this should be transparent enough, I think. I.e. the dogonlanguages community on Zenodo will have all output **or** persistent links to the outputs.

**Comment by bambooforest** [comment]
@xrotwang 

https://github.com/dogonlanguages/bibliography

Not sure what this file is and if you need something like it:

https://github.com/tupian-language-resources/bibliography/blob/main/main.tex

**Comment by bambooforest** [comment]
cf. https://github.com/clld/dogonlanguages-data/issues/19

**Comment by xrotwang** [comment]
Maybe `main.tex` is for creating a PDF bibliography (correct, @LinguList  ?). Don't think we need this here.

**Comment by xrotwang** [comment]
On second thought, if we want to release this to Zenodo, maybe a PDF would be a nice addition. But then we should add a `RELEASING.md` with release instructions, too :)

**Comment by bambooforest** [comment]
Yeah I can create a tex that simply dumps the entire biblio as PDF. Do you have an example of proper RELEASING?

**Comment by xrotwang** [comment]
Here's an example from tudet: https://github.com/tupian-language-resources/tudet/blob/main/RELEASING.md

**Comment by LinguList** [comment]
@xrotwang, sorry for late answer, but the main.tex is used on overleaf to guarantee that people can see the consequences of their bibtex decisions. This is crucial for those who are less firm in Bibtex. AND, having the bibliography in PDF is also a nice bonus.

**Comment by LinguList** [comment]
Since compilation of the bibtex file into a PDF is something like a test, it also spares us from making a test independently and then tellling users they should adjust the latex code. This is at least why we did this with TULAR.
